packages/xo-web/src/xo-app/xostor/new-xostor-form.js

```diff
@@ -74,7 +74,7 @@ const StorageCard = decorate([
   injectState,
   ({ effects, state }) => (
     <Card>
-      <CardHeader>{_('storage')}</CardHeader>
+      <CardHeader>{_('generalTabName')}</CardHeader>
```

Comment:
The `storage` message doesn't seem to be used anymore. Remove it if this is the case.